dts/bindings/serial/realtek,rts5912-uart.yaml

```diff
@@ -22,3 +22,22 @@ properties:
 
   pinctrl-names:
     required: true
+
+  rx-gpios:
+    description: |
+      Select the wake up pin for uart rx,
+      gpioa = 0 ~ 15,
+      gpiob = 16 ~ 31,
+      gpioc = 32 ~ 47,
+      gpiod = 48 ~ 63,
+      gpioe = 64 ~ 79,
+      gpiof = 80 ~ 95,
+      gpiog = 96 ~ 111,
+      gpioh = 112 ~ 127,
+      gpioa = 128 ~ 131
+    type: phandle-array
```

Comment:
Realtek landed macros to translate the pin name to the GPIO port and pin number.
https://github.com/zephyrproject-rtos/zephyr/blob/73ea9419c354526edf175607c5f0713f892a5196/include/zephyr/dt-bindings/gpio/realtek-gpio.h#L57

Please update the help text to use the new macros. For example:
```
   rx-gpios = <RTS5912_GPIO013 0>;
```
But I recommend you select a GPIO that can be used as a UART RX pin.